### PR TITLE
Remove references to IBM Q from the README.md

### PR DESCRIPTION
**Pull request by 1ucian0**
This PR update the `README.md`:
 * Removes references to IBM Q in favor of IBM Quantum
 * Removes the notion of "element"
 * Updates the way to get the token in IBM (with images)
 * Some other minor typos.

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

### CODE REVIEW

File: README.md
Comment (Cryoris) [inline README.md:6]: Do we want to add a sentence saying that e.g. the applications and other packages are built on it, or just leave as is?

Comment (1ucian0) [reply README.md:6]: I would prefer leaving it as it is for now.

**Review by Cryoris — COMMENTED** [review]
There's still one occurance of "disk" (instead of "local") on L70 if you want to update that 🙂

**Comment by 1ucian0** [comment]
good catch. Fixed in 93a809857. I really dont like this "disk" terminology... sounds soo "hardwareish" :P

File: README.md
Comment (jakelishman) [inline README.md:6]: If we're going to give the full package name in code format, it might not hurt to say that it supplies the `qiskit` namespace - I think this is a fairly common source of confusion for new users (and experienced ones, sometimes).  Alternatively, we could avoid `` `qiskit-terra` `` and just write "Terra", similar to how we had it before?

File: README.md
Comment (jakelishman) [inline README.md:54]:
```suggestion
**real quantum chip**.
```

File: README.md
Comment (jakelishman) [inline README.md:55]:
```suggestion
In order to do so, you need to configure Qiskit with credentials for using a quantum computer cloud provider. For example, to connect it to your IBM Quantum account:
```

File: README.md
Comment (jakelishman) [inline README.md:79]: I think the former is clearer in this situation; "locally" can also mean "in memory" and you'd have to get the intent from the rest of the context.

File: README.md
Comment (jakelishman) [inline README.md:62]: You probably want to put a bit of horizontal space between these two images as well.

If possible, it would be neater if we could rescreenshot the two to:
- avoid the white background encroaching in the account dropdown
- make both images the same natural height (i.e. the same number of pixels tall), even if their widths are a bit different.

They'll sit next to each other on the page a bit more neatly if so.  These images also won't show up on the PyPI page, and they might be a bit temperamental on GitHub too because of the relative address.  One trick I've used in the past is to upload the images into a random issue, then use the permalink to those images in the readme instead.  The images don't end up checked into the repo, but the only way anyone would really see them anyway is on some online rendered version.

As an example, that's how the images in [this repository's readme](https://github.com/openqasm/openqasm.vim) are hosted - they're in openqasm/openqasm#239 (it was before we split the organisation off, so everything was still in a monorepo then).

File: README.md
Comment (jakelishman) [reply README.md:79]: Oh, I just saw above that you and Julien were talking about this.  I still think it's worth using "disk" rather than locally here, because the next line is really trying to distinguish persistent and volatile storage.

File: README.md
Comment (1ucian0) [reply README.md:6]: good point.
```suggestion
The core component of Qiskit is **Terra**.
```

**Comment by 1ucian0** [comment]
The screenshot, following the idea in https://github.com/Qiskit/qiskit-terra/pull/8686#discussion_r965332446

ibm_account

File: README.md
Comment (1ucian0) [reply README.md]: updated in 5cb64e623

File: README.md
Comment (1ucian0) [reply README.md:80]: I agree with the emphasis on "persistency". So changed in 38690e56f7b19b07d245fd7ccc859fd6766ef156

File: README.md
Comment (mtreinish) [inline README.md:55]: For this section I'd actually make it more general and less IBM specific at the beginning and only mention IBM as an example. Something like:

```suggestion
Qiskit enables you to run code on real quantum computers. To to do this it provides an abstraction
layer that lets hardware vendors build packages that provde an interface to their hardware from Qiskit.
Using these ``providers`` you can run any Qiskit code against a real quantum computer.

For example, to use Qiskit with IBM Quantum's cloud quantum computing service:
```

File: README.md
Comment (mtreinish) [inline README.md:49]: We're probably going to remove the ibmq examples directory so we can probably just delete this line

File: README.md
Comment (mtreinish) [inline README.md:24]: If we're going to mention basicaer here I would call out that it's very limited and different from qiskit-aer. If you want to do real simulation work you should use qiskit-aer as basicaer is just for testing.

File: README.md
Comment (mtreinish) [inline README.md:83]: Should we really have this in the qiskit-terra readme? Especially with the state of flux around ibm provider packages maybe we should just link to the readme of the provider package du jour and as we shouldn't want to worry about how to setup authentication or credentials as part of qiskit terra.

Comment (1ucian0) [reply README.md:87]: I agree. However, let's keep it for now until the provider README gets released and updated.

File: README.md
Comment (1ucian0) [reply README.md:24]: done in 7055ed04ad0746801d4236a03ac545d2b5ee3774

File: README.md
Comment (1ucian0) [reply README.md]: done in 9296ac8610054488a2858d2bf508d51778a7e26a

**Comment by mtreinish** [comment]
In order to speed up the review cycle on this a bit, instead of inline suggestions I pushed this PR to your branch: https://github.com/1ucian0/qiskit-terra/pull/30 with some changes I think we should make to the readme while we're refactoring. I'm good to merge this once that is reviewed and merges (although we probably should get a second review as I made some major changes in there).

File: README.md
Comment (kdk) [inline README.md:81]:
```suggestion
Qiskit provides an abstraction layer that lets users run quantum circuits on hardware from any
vendor that provides an interface to their systems through Qiskit. Using these ``providers`` you can run any Qiskit code against
real quantum computers. Some examples of published provider packages for running on real hardware are:
```

File: README.md
Comment (mtreinish) [reply README.md]: I think it's probably better to leave it as `BasicAer` just to have this example continue working after aer is removed from the metapackage. If we didn't we'd basically have to say do `pip install qiskit qiskit-aer` to run this example

**Review by kdk — APPROVED** [review]

**Pull request merged**
